assignments/Assign10CDogFactsAPI.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Dog Breeds</title>
+</head>
+<body>
+    <header>
+        <h2>List of dog breeds</h2>
+        <button id="listBreeds">List Breeds</button>
```

Comment:
You'll have to change this to `<input type="button" ...>` to avoid the page reload problem.